README.adoc

```diff
@@ -25,6 +25,8 @@ is set, for example by running `export GO15VENDOREXPERIMENT=1`.
 In Go `1.6` it is enabled by default and in Go `1.7` it is always enabled
 without the ability to turn it off.
 
+If on Windows, the `GOPATH` environment variable is not set by default - see http://www.wadewegner.com/2014/12/easy-go-programming-setup-for-windows/[step 5].
```

Comment:
this is not really dependent on windows. Same goes for any GO project. You need to setup a $GOPATH for all.